mobile/src/main/java/ch/epfl/sdp/mobile/ui/play/PlayScreen.kt

```diff
@@ -1,49 +1,93 @@
-import androidx.compose.material.ExtendedFloatingActionButton
+import androidx.compose.material.Divider
 import androidx.compose.material.Icon
+import androidx.compose.material.MaterialTheme
 import androidx.compose.material.Text
 import androidx.compose.runtime.Composable
 import androidx.compose.ui.Alignment
 import androidx.compose.ui.Modifier
 import androidx.compose.ui.unit.dp
 import ch.epfl.sdp.mobile.state.LocalLocalizedStrings
 import ch.epfl.sdp.mobile.ui.Add
-import ch.epfl.sdp.mobile.ui.PawniesColors
+import ch.epfl.sdp.mobile.ui.LocalPlay
+import ch.epfl.sdp.mobile.ui.OnlinePlay
 import ch.epfl.sdp.mobile.ui.PawniesIcons
 
 /**
  * Composable that composes the PlayScreen [TODO] Contains a new game button only, should be
  * expanded to include history of matches
  * @param state state of the PlayScreen
  * @param modifier the [Modifier] for this composable.
+ * @param contentPadding The [PaddingValues] to apply to the content
  */
 @Composable
 fun PlayScreen(
     state: PlayScreenState,
     modifier: Modifier = Modifier,
     contentPadding: PaddingValues = PaddingValues(),
 ) {
-  Box(modifier = modifier.fillMaxSize().padding(contentPadding)) {
-    NewGameButton(onNewGame = state.onNewGameClick, modifier = Modifier.align(Alignment.BottomEnd))
+  val buttonState = rememberExpandableFloatingActionButtonState()
+
+  Box(
+      modifier = modifier.fillMaxSize().padding(contentPadding),
+      contentAlignment = Alignment.BottomEnd) {
+    NewGameButton(
+        state = buttonState,
+        onLocalGame = {
+          state.onLocalGameClick()
+          buttonState.expanded = false
+        },
+        onRemoteGame = {
+          state.onOnlineGameClick()
+          buttonState.expanded = false
+        },
+        modifier = Modifier,
+    )
   }
 }
 
 /**
  * Composable that composes a new game button
- * @param onNewGame callable upon button action
+ * @param state the [ExpandableFloatingActionButton]'s state
+ * @param onLocalGame callable upon local game button press
+ * @param onRemoteGame callable upon remote game button press
  * @param modifier the [Modifier] for this composable.
  */
 @Composable
-fun NewGameButton(onNewGame: () -> Unit, modifier: Modifier = Modifier) {
+fun NewGameButton(
+    state: ExpandableFloatingActionButtonState,
+    onLocalGame: () -> Unit,
+    onRemoteGame: () -> Unit,
+    modifier: Modifier = Modifier,
+) {
   val strings = LocalLocalizedStrings.current
-  ExtendedFloatingActionButton(
-      text = { Text(strings.newGame) },
-      onClick = onNewGame,
-      shape = CircleShape,
-      modifier = modifier.padding(16.dp).height(56.dp),
-      backgroundColor = PawniesColors.Green800,
-      icon = { Icon(PawniesIcons.Add, null) })
+
+  ExpandableFloatingActionButton(
+      state = state,
+      expandedContent = {
+        ExpandableFloatingActionButtonItem(
+            onClick = onLocalGame,
+            icon = { Icon(PawniesIcons.LocalPlay, null) },
```

Comment:
You should (import) and use the `Image` composable here, to preserve the colors.

```suggestion
            icon = { Image(PawniesIcons.LocalPlay, null) },
```

Reply:
Fixed in [e417a59](https://github.com/epfl-SDP/android/pull/226/commits/e417a59720326823af3fab8f8d4dbed70cff13db)